### PR TITLE
Discovery fixes

### PR DESCRIPTION
**Pull request by seanyesmunt**

### CODE REVIEW

File: dist/bundle.es.js [hunk 1]
```diff
@@ -872,32 +872,37 @@ function parseURI(URI, requireProto = false) {
 
   // Validate protocol
   if (requireProto && !proto) {
-    throw new Error(__('LBRY URIs must include a protocol prefix (lbry://).'));
+    console.error(__('LBRY URIs must include a protocol prefix (lbry://).'));
```
Comment (kauffj) [inline dist/bundle.es.js]: Why is this better? Won't anything using this choke on receiving an empty object anyway?

File: src/redux/selectors/claims.js [hunk 2]
```diff
@@ -256,12 +256,16 @@ export const selectMyClaims = createSelector(
...
+  myClaims =>
+    myClaims.filter(claim => !claim.name.match(/^@/)).sort((a, b) => a.timestamp - b.timestamp)
 );
 
 export const selectMyClaimUrisWithoutChannels = createSelector(
   selectMyClaimsWithoutChannels,
-  myClaims => myClaims.map(claim => `lbry://${claim.name}#${claim.claim_id}`)
+  myClaims =>
+    myClaims
+      .sort((a, b) => a.confirmations - b.confirmations)
```
Comment (kauffj) [inline src/redux/selectors/claims.js]: It's a corner case, but there's nothing stopping two publishes from being in the same block.

Comment (kauffj) [reply src/redux/selectors/claims.js]: Particularly with YT sync publishes, this is likely.

File: src/redux/selectors/file_info.js [hunk 3]
```diff
@@ -232,5 +232,9 @@ export const selectFileListDownloadedSort = createSelector(
 export const selectDownloadedUris = createSelector(
   selectFileInfosDownloaded,
   // We should use permament_url but it doesn't exist in file_list
-  info => info.map(claim => `lbry://${claim.claim_name}#${claim.claim_id}`)
+  info =>
+    info
+      .slice()
+      .reverse()
+      .map(claim => `lbry://${claim.claim_name}#${claim.claim_id}`)
```
Comment (kauffj) [inline src/redux/selectors/file_info.js]: Is the comment here still true, that the URL isn't available on the claim?

Comment (seanyesmunt) [reply src/redux/selectors/file_info.js]: Yep

File: dist/bundle.es.js (hunk 1 above)
Comment (seanyesmunt) [reply dist/bundle.es.js]: I guess they could, probably better to just fix it in the app. I'll remove

File: src/redux/selectors/claims.js (hunk 2 above)
Comment (seanyesmunt) [reply src/redux/selectors/claims.js]: Yep, I should have been using `timestamp`

**Comment by seanyesmunt** [comment]
@kauffj Going to merge this so I can merge the `lbry-desktop` pr (aka I already merged that because I forgot this PR needed to be merged too)

Let me know if you have more comments.

**Pull request merged**